frontend/src/__tests__/cypress/cypress/utils/oc_commands/project.ts

```diff
@@ -31,25 +31,34 @@ export const createOpenShiftProject = (
  * Delete an OpenShift Project given its name
  *
  * @param projectName OpenShift Project name
+ * @param options Configuration options for the delete operation
+ * @param options.timeout Timeout in milliseconds for the command (only used when wait is true)
+ * @param options.wait Whether to wait for the deletion to complete (default: true)
  * @returns Result Object of the operation
  */
 export const deleteOpenShiftProject = (
   projectName: string,
-  options: { timeout?: number } = {},
+  options: { timeout?: number; wait?: boolean } = {},
 ): Cypress.Chainable<CommandLineResult> => {
-  const { timeout } = options;
-  const ocCommand = `oc delete project ${projectName}`;
-  return cy
-    .exec(ocCommand, { failOnNonZeroExit: false, ...(timeout && { timeout }) })
-    .then((result) => {
-      if (result.code !== 0) {
-        cy.log(`ERROR deleting ${projectName} Project
+  const { timeout, wait = true } = options;
```

Comment:
```suggestion
  const { timeout, wait = false } = options;
```
Why not default this to false -- waiting for OpenShift Project deletions seems a bit weird; maybe a given test would care because of some watching delete functionality -- but I can't imagine this is important and just gives a quirk to the timeout need

Reply:
The `oc delete` project command has the `wait` value set to True by default, so I wanted to mimic that behavior by making the developer consciously choose to set it to False if they want to.
Also, I think setting it to False by default could have negative consequences, since I’ve only updated the cases where I know it matters (like in the test teardowns), but I haven’t reviewed every single place where that function is called.
On top of that, if at some point calling oc delete returns an error or an unexpected value, it won’t be easy to debug.

Reply:
I'm not sure I agree with that assessment, but this is in your wheelhouse -- I'll retract my request.